src/EmailCampaigns/PostsEmailDigest/PostsEmailDigest.php

```diff
@@ -111,7 +111,14 @@ public function post_collection($email_campaign_id)
                 'after'  => $newer_than_timestamp
             ]
         ];
-
+    
+        //apply post translation filter
+        $parameters = apply_filters('mailoptin_post_digest_translated_post_args',
```

Comment:
@ShoboySnr why do you need to add this filter?

Why not use mo_post_digest_get_posts_args filter instead?